linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_csh_cshrc/ignition/shared.yml

```diff
@@ -0,0 +1,18 @@
23%20csh%20configuration%20for%20all%20shell%20invocations.%0A%0A%23%20By%20default%2C%20we%20want%20this%20to%20get%20set.%0A%23%20Even%20for%20non-interactive%2C%20non-login%20shells.%0A%23%20Current%20threshold%20for%20system%20reserved%20uid/gids%20is%20200%0A%23%20You%20could%20check%20uidgid%20reservation%20validity%20in%0A%23%20/usr/share/doc/setup-%2A/uidgid%20file%0Aif%20%28%24uid%20%3E%20199%20%26%26%20%22%60id%20-gn%60%22%20%3D%3D%20%22%60id%20-un%60%22%29%20then%0A%20%20%20%20umask%20{{{ var_accounts_user_umask }}}%0Aelse%0A%20%20%20%20umask%20{{{ var_accounts_user_umask }}}%0Aendif%0A%0Aif%20%28%24%3Fprompt%29%20then%0A%20%20if%20%28%24%3Ftcsh%29%20then%0A%20%20%20%20set%20promptchars%3D%27%24%23%27%0A%20%20%20%20set%20prompt%3D%27%5B%25n%40%25m%20%25c%5D%25%23%20%27%0A%20%20%20%20%23%20make%20completion%20work%20better%20by%20default%0A%20%20%20%20set%20autolist%0A%20%20else%0A%20%20%20%20set%20prompt%3D%5C%5B%24user%40%60hostname%20-s%60%5C%5D%5C%24%5C%20%0A%20%20endif%0Aendif%0A%0Aif%20%28%20%24%3Ftcsh%20%29%20then%0A%09bindkey%20%22%5E%5B%5B3%7E%22%20delete-char%0Aendif%0A%0Abindkey%20%22%5ER%22%20i-search-back%0Aset%20echo_style%20%3D%20both%0Aset%20histdup%20%3D%20erase%0Aset%20savehist%20%3D%20%281024%20merge%29%0A%0Aif%20%28%24%3Fprompt%29%20then%0A%20%20if%20%28%24%3FTERM%29%20then%0A%20%20%20%20switch%28%24TERM%29%0A%20%20%20%20%20%20case%20xterm%2A%3A%0A%20%20%20%20%20%20%20%20if%20%28%24%3Ftcsh%29%20then%0A%09%20%20set%20prompt%3D%27%25%7B%5C033%5D0%3B%25n%40%25m%3A%25c%5C007%25%7D%5B%25n%40%25m%20%25c%5D%25%23%20%27%0A%20%20%20%20%20%20%20%20endif%0A%20%20%20%20%20%20%20%20breaksw%0A%20%20%20%20%20%20case%20screen%3A%0A%20%20%20%20%20%20%20%20if%20%28%24%3Ftcsh%29%20then%0A%20%20%20%20%20%20%20%20%20%20set%20prompt%3D%27%25%7B%5C033k%25n%40%25m%3A%25c%5C033%5C%5C%25%7D%5B%25n%40%25m%20%25c%5D%25%23%20%27%0A%20%20%20%20%20%20%20%20endif%0A%20%20%20%20%20%20%20%20breaksw%0A%20%20%20%20%20%20default%3A%0A%20%20%20%20%20%20%20%20breaksw%0A%20%20%20%20endsw%0A%20%20endif%0Aendif%0A%0Asetenv%20MAIL%20%22/var/spool/mail/%24USER%22%0A%0A%23%20Check%20if%20we%20aren%27t%20a%20loginshell%20and%20do%20stuff%20if%20we%20aren%27t%0Aif%20%28%21%20%24%3Floginsh%29%20then%0A%20%20%20%20if%20%28%20-d%20/etc/profile.d%20%29%20then%0A%20%20%20%20%20%20%20%20set%20nonomatch%0A%20%20%20%20%20%20%20%20foreach%20i%20%28%20/etc/profile.d/%2A.csh%20%29%0A%20%20%20%20%20%20%20%20%20%20%20%20if%20%28%20-r%20%22%24i%22%20%29%20then%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20if%20%28%24%3Fprompt%29%20then%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20source%20%22%24i%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20else%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20source%20%22%24i%22%20%3E%26/dev/null%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20endif%0A%20%20%20%20%20%20%20%20%20%20%20%20endif%0A%20%20%20%20%20%20%20%20end%0A%20%20%20%20%20%20%20%20unset%20i%20nonomatch%0A%20%20%20%20endif%0Aendif%0A
```

Comment:
When the content is built `{{{ var_accounts_user_umask }}}` is rendered empty, like this: `%20umask%20%0Aelse`.

I think the correct value is:
`<xccdf-1.2:sub idref="xccdf_org.ssgproject.content_value_var_accounts_user_umask" use="legacy"/>`
which will be substituted by the scanner when rendering out the fix snippet.

Reply:
changed, thanks